Assets/AnkrSDK/Runtime/Data/NetworkNameExtension.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel;
+
+namespace AnkrSDK.Data
+{
+    public static class NetworkNameExtension
+    {
+        public static int GetNetworkChainId(this NetworkName networkName)
```

Comment:
Instead of hardcoding chainid into an extension you can just specify enum values for each NetworkName

Reply:
Okay, but it will require a comment to not change enum values